scalardl/project.clj

```diff
@@ -28,4 +28,5 @@
              :default [:base :system :user :provided :dev :use-released]}
   :java-source-paths ["contract"]
   :main scalardl.runner
-  :aot :all)
+  :aot :all
+  :pedantic? false)
```

Comment:
Actually, I'm not very sure the root cause of the issue, but it looks like we are hitting the same issue in the following article, and the solution is adding `:pedantic? false` in `project.clj`.
https://github.com/technomancy/leiningen/issues/2763